geopy/geocoders/__init__.py

```diff
@@ -91,7 +92,7 @@
     'Yandex',
     "What3Words",
     "Photon",
-    "Mapzen",
+    "Mapzen"
```

Comment:
don't delete the trailing comma. It is there on purpose, so adding or re arranging the tuple is easier and cleaner